core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -82,6 +82,12 @@ public static GeneticProfile loadGeneticProfile(File file) throws IOException, D
                throw new RuntimeException("Error: genetic_profile record found with same Stable ID as the one used in your data:  "
                        + existingGeneticProfile.getStableId() + ". Remove the existing genetic_profile record first.");
             } else if (geneticProfile.getDatatype().equals("FUSION")) {
+                // populate the structural variant genetic profile for fusions
+                GeneticProfile gp = new GeneticProfile(geneticProfile);
+                gp.setGeneticAlterationType(GeneticAlterationType.STRUCTURAL_VARIANT);
+                gp.setStableId(gp.getStableId().replace("mutations","fusion"));
```

Comment:
I guess this relies on this part? https://github.com/cBioPortal/cbioportal/blob/7ae3828ac87d2f88703d8f9f3f60d5d756085a4b/core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java#L290-L301

it's a little funky but I guess we can remove all the `FUSION` conditional once we remove `FUSION` data and only have `STRUCTURAL_VARIANT` GeneticAlterationType:

https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/java/org/mskcc/cbio/portal/model/GeneticAlterationType.java#L42-L43

Reply:
That is right, @inodb, till we completely deprecate the Fusion genetic type, it might take a while.